index.html

```diff
@@ -173,30 +173,107 @@ <h2>
     </section>
     <section>
       <h2>
-        <code>pointerlockchange</code> and <code>pointerlockerror</code> Events
+        Pointer Lock and Interfaces
       </h2>
-      <p>
-        Two events are used to communicate pointer lock state change or an
-        error in changing state. They are named <dfn>pointerlockchange</dfn>
-        and <dfn>pointerlockerror</dfn>. If pointer lock is entered or exited
-        for any reason a <a>pointerlockchange</a> event must be sent.
-      </p>
-      <p>
-        <a>User agents</a> must deliver these events by <a data-lt=
-        "queue a task">queuing a task</a> to <a>fire an event</a> of the
-        appropriate name with its <code>bubbles</code> attribute [[DOM]] set to
-        false to the pointer lock <a>target</a> element's [=Node/node
-        document=].
-      </p>
-      <p class="note">
-        Magnification software increases the size of content on the screen. It
-        uses the mouse to move the magnified point of focus around. When a
-        pointer lock is initiated, the magnification software needs to switch
-        to using the keyboard to move the magnified point of focus around
-        instead. When a pointerlockchange event is fired, web browsers
-        therefore need to make sure the event is communicated to assistive
-        technologies like screen magnifiers.
-      </p>
+      <section>
+        <h3>
+          The <dfn>Pointer Lock State</dfn> Definition
+        </h3>
+          <p><a>Pointer Lock State</a> is the state where a single DOM element, which
+            we will call the <dfn>target</dfn>, receives all mouse events and the cursor
```

Comment:
This should state that the [user agent](https://infra.spec.whatwg.org/#user-agent) has a pointer-lock target, to be clear this is scoped to the entire UA and not, e.g., to each Document.

And it should be stated that the target is either an `Element` or null.

I think the name "pointer-lock target" is also a bit clearer than just "target".

Finally, you need to improve the spec text so that everything which locks/unlocks explicitly changes the user agent's pointer-lock target, either to an element or to null. That includes requestPointerLock(), exitPointerLock(), and the user dismissing the pointer lock with no API calls. To do this, it may help to have a central definition for **enter pointer lock** and **exit pointer lock** which: put the UA in/out of the pointer lock state, then queue a task to manage the pointer-lock target and fire the pointerlockchange event at the target's node document.